client/src/elements/schema-editor/schema-editor-geojson-point.js

```diff
@@ -44,12 +44,12 @@ export class SchemaEditorGeojsonPoint {
       return;
     }
 
-    mapboxgl.accessToken = schemaEditorConfig.geojson.layer.accessToken;
```

Comment:
can we keep the mapbox token support somehow? so users without their own tile service can still configure it?

Reply:
Fixed in https://github.com/nzzdev/Q-editor/pull/226/commits/f1b377d735840ed082e674d884587347cae614fb